coalesce/users/models.py

```diff
@@ -9,6 +9,8 @@
 
 class User(AbstractUser):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    first_name = models.TextField()
+    last_name = models.TextField()
```

Comment:
Please run `docker-compose exec api ./manage.py makemigrations` and include the auto-generated migration file in this pull request. That will ensure that existing database tables are updated when this code lands.

Reply:
Ok will do